packages/KModal/KModal.vue

```diff
@@ -5,39 +5,43 @@
     class="k-modal"
     role="dialog">
     <div
-      class="modal-backdrop"
+      class="k-modal-backdrop modal-backdrop"
       @click="close">
 
       <div
-        class="modal-dialog"
+        class="k-modal-dialog modal-dialog"
         @click.stop>
-        <div class="modal-content">
+        <div class="k-modal-content modal-content">
           <div
-            :class="helpText ? 'mb-3' : 'mb-4'"
-            class="modal-header"
+            :class="$scopedSlots.help || helpText ? 'mb-4' : 'mb-5'"
+            class="k-modal-header modal-header"
           >
             <slot name="header-content">{{ title }}</slot>
           </div>
           <div
             v-if="$scopedSlots.help || helpText"
-            class="modal-help">
+            class="k-modal-help modal-help">
             <slot name="help">{{ helpText }}</slot>
           </div>
-          <div class="modal-body">
+          <div class="k-modal-body modal-body">
             <slot name="body-content">{{ content }}</slot>
           </div>
-          <div class="modal-footer">
+          <div class="k-modal-footer modal-footer d-flex">
             <slot name="footer-content">
-              <KButton
-                :appearance="actionButtonAppearance"
-                @click="proceed">
-                {{ actionButtonText }}
-              </KButton>
               <KButton
```

Comment:
this should have a keyboard event, like onEnter

Reply:
Using `@keyup.esc`, thanks for the idea!!